### PR TITLE
Contact Form - Portfolio #4

### PR DESCRIPTION
**Pull request by sheylaPozo**
## For the third milestone of your portfolio website we worked on

- Adding a contact form to my portfolio and the portfolio of my coding partner.
- Creating forms with HTML5 validations.
- Understanding the importance of UX.
- Understanding how to parse a Figma design to create a UI.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -174,5 +174,43 @@ <h3 class="card-title">TONIC</h3>
...
+          placeholder="Email"
+        />
+        <textarea
+          name="message"
+          id="message"
+          required
+          placeholder="Write your message here"
+          maxlength="500"
+        >
+        </textarea>
```
Comment (Temmarie) [inline index.html]: - [x] You need to stick to the design templates in Figma as much as possible, this includes the font, colours, images and spacing (margin/padding) 
- This section is missing the necessary background image.
- You are also missing the placeholder for the message.
- See both images below and fix the differences, take note of the font used in the placeholders.

**Original** | **Your clone**
  ----------- | -----------
  image | image

**Review by Temmarie — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED! :recycle:

### Hi @sheylaPozo and @chaw-bot     ! :wave: :wave:

#### You have done really well in implementing the requirements.
- You used proper GitHub Flow ✔️ 
- Your linter checks are all passing ✔️ 
- Your README and PR message are descriptive ✔️ 

 However, there are changes required for you to make in order for this project to be approved. 

> Keep in mind that your long hours of work are appreciated. However, the code review process is for improving the code you wrote, and not for criticizing it. :smile:

- - -

### **Required Changes** ♻️
_See the inline comments below for more required changes_


### Optional Changes 👌
Every comment with the OPTIONAL prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend that you take them into account as they can make your code better.
- Consider making your commits more descriptive and written in the imperative tense. See [here](https://chris.beams.io/posts/git-commit/) for more context.

_If anything doesn't seem clear to you, feel free to leave your question or comment in the PR thread below_

**_"Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise."_**
- - - 
**Note that you still have 2 more reviews as written in the [Code Reviews Limit Policy](https://microverse.zendesk.com/hc/en-us/restricted?return_to=https%3A%2F%2Fmicroverse.zendesk.com%2Fhc%2Fen-us%2Farticles%2F1500004088561), however, if you feel this review is unfair, you can request a second opinion using [this](https://airtable.com/shrQAqnBwek5a0O0s) form.**

__Kindly make the necessary changes and submit another code review request.__

File: index.html (hunk 1 above)
Comment (sheylaPozo) [reply index.html]: Fixed required changes, thank you.

File: index.html [hunk 2]
```diff
@@ -1,178 +1,236 @@
...
+            name="user-email"
+            id="user-email"
+            required
+            placeholder="Email"
+          />
+          <textarea
+            name="message"
+            id="message"
+            placeholder="write your message here"
+            maxlength="500"></textarea>
```
Comment (Davidosky007) [inline index.html:228]: [OPTIONAL]
I strongly suggest you add " required" attribute to activate the validation on submit for this input field. Currently, the form submits even when nothing is entered in the input field.

**Review by Davidosky007 — APPROVED** [review]
Hi @sheylaPozo @chaw-bot, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉
CR1 GIF 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. 
------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (Davidosky007) [reply index.html]: done 👍

File: index.html (hunk 2 above)
Comment (sheylaPozo) [reply index.html:228]: okii will do :)

**Pull request merged**